apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -1109,6 +1149,48 @@ public void SilentlyFixedParameters()
             Assert.Equal(TimeSpan.FromSeconds(59), sub4.AutoExtendInterval);
         }
 
+        [Fact]
+        public void ExactlyOnceDelivery_SilentlyFixedParameters()
```

Comment:
Hmm, I think this test should be split in plenty (and same for the one above from where you copied this one, sorry).

Reply:
Done.